example_cp.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
```

Comment:
@d33bs Not sure what this is referencing here. I will inquire to you about this next time you are in :smile:

Reply:
@jenna-tomkinson, apologies about the confusion here 🙂 . This comment was intended more for the file itself rather than line-level. For example, would it make sense for `example_cp.py` to be placed under the directory `example_files/` in some way (along with the others)?

Reply:
Interesting point! I think that Roshan uses it for any of the files that are used within the notebooks. I think it makes sense to keep files used to run idrstream separate from the notebook so maybe it would be good to make a `notebooks/` folder.

@d33bs what do you think about this?

Reply:
Sounds like a good idea; do what's most useful to the audience who may make use of the content 👍 .

Reply:
Hmmmm... originally, the `example_files` folder had DP files and everything else necessary for the one example notebook. But now that we have so many examples (DP/CP notebook and DP/CP py), I think those 4 example files should go into `examples/` and everything currently in `example_files` should be moved to `examples/DP_example_files/'.

@jenna-tomkinson What are your thoughts?